serdes-ballerina/serdes.bal

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# SerDes object that can be used to create custom serializers/deserializers.
+public type SerDes object {
+
+    public isolated function serialize(anydata data) returns byte[]|Error;
+
```

Comment:
```suggestion
```

Reply:
This line still there. Do not add a line between two abstract function definitions.